libcodechecker/analyze/plist_parser.py

```diff
@@ -82,7 +82,9 @@ def parse_plist(path):
     reports = []
     files = []
     try:
-        plist = plistlib.readPlist(path)
+        plist = None
```

Comment:
Are you sure you need this?

Reply:
We checked the code in plistlib and we thought if the `parse` method is failing it would not close the file.
```python
# plistlib.py
def readPlist(pathOrFile):
    didOpen = 0
    if isinstance(pathOrFile, (str, unicode)):
        pathOrFile = open(pathOrFile)
        didOpen = 1
    p = PlistParser()
    rootObject = p.parse(pathOrFile)
    if didOpen:
        pathOrFile.close()
    return rootObject
```
I have parsed a wrong formatted plist file and I saw that this code closed the file properly. So my modification is unnecessary.

Reply:
I mean, I am not sure that you need to introduce plist upfront outside of the with block.